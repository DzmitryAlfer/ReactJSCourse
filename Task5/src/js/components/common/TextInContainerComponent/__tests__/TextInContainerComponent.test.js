import React from 'react';
import Renderer from 'react-test-renderer';
import {TextInContainerComponent} from "../";

jest.mock('../../TextComponent');

describe('TextInContainerComponent', () => {
    test('Component render', () => {

        const tree = Renderer.create(
            <TextInContainerComponent text='some-text' textClass='text-class' containerClass='container-class'/>
        ).toJSON();

        expect(tree).toMatchSnapshot();
    });
});