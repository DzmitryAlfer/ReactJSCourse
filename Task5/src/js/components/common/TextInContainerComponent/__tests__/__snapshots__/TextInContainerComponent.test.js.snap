// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInContainerComponent Component render 1`] = `
<div
  className="container-class"
>
  <span>
    TextComponent MOCK
  </span>
</div>
`;
